### PR TITLE
Tenon.io accessibility improvements 

### LINKED ISSUE #469: Tenon.io accessibility findings
**Description of the issue:**

The following issues were found when running an accessibility test against Website Honestly

- [x] Social media links in the footer are missing link text throughout the site (fixing this would improve our accessibility score by quite a bit)

- [x] Remove duplicate ids, especially if they identify a control. 'Symbols' and 'arrow' are being used repeatedly. This looks to be especially bad on the 'About us' page

- [x] Some images are still missing alt text, e.g. img role="presentation class="styles__leanImage___7Cp9U" src="/assets-honestly/lean-2WF10.png">'

- [x] Duplicate alt-text on the Retailer case study page, img src="/assets-honestly/header-3SKLJ.jpg" alt="shopping cart" class="style__header__image___1GrxS"

- [x] Empty buttons on the About Us page (I think in the History Slice)

- [x] Alt text for some images used in social media may be a bit too long (need to be concise, yet informative

- [x] Same alt text on images with different src attributes - these should be distinct

- [x] Alt text should never consist only of numerics, e.g. '2012' on the 'About us' page

- [x] On the 'About us' page, the link text in this element is too uninformative a rel="noopener noreferrer" target="_blank" href="/services/technology">here</a

~- [ ] A lot of missing alt text and missing text in links on the blog pages. Not entirely sure what we can do about this, given the links and images are hosted in Squarespace~ Should be taken care of in a separate blog ticket

### PR DESCRIPTION
**Pull request by martinhartt**
### Motivation

Fixes #469 

### Pre-merge checklist

- [ ] Documentation
- [ ] Unit tests
- [ ] Reviews
  - [ ] Code review
  - [ ] Design review
- [ ] Manual testing
  - [ ] Windows 7 IE 11
  - [ ] Windows 10 Edge
  - [ ] Mac OS El Capitan Safari
  - [ ] Mac OS El Capitan Chrome
  - [ ] Mac OS El Capitan Firefox
  - [ ] iOS 9 Safari
  - [ ] Android 6 Chrome
  - [ ] Listen to the site on a screen reader
  - [ ] Navigate the site using the keyboard only
- [ ] Tester approved

### CODE REVIEW

**Comment by martinhartt** [comment]
After messaging the people behind tenon.io, I discovered that the remaining issues regarding `Same ALT text on images with different src attributes` are bugs on their side. Once they fix these, we should have 0 issues.

**Comment by gregdorward** [comment]
@martinhartt I'm seeing an error on the tool I'm using, where it's reported that "A form control has more than one label associated with it.". I think it's talking about the desktop header menu items. Is this because we label them for desktop and mobile? I'm guessing we may have to live with this one?

**Comment by gregdorward** [comment]
screen shot 2017-08-03 at 12 16 49

File: site/pages/our-work/case-study/ft/index.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const FinancialTimesCaseStudy = ({ contactUsURL }: CaseStudyFinancialTimesProps)
     <div className={styles.header}>
       <div className={styles.header__container}>
         <div className={styles.header__imageContainer}>
-          <img src={headerImage} alt="shopping cart" className={styles.header__image} />
+          <img src={headerImage} alt="meeting" className={styles.header__image} />
```
Comment (gregdorward) [inline site/pages/our-work/case-study/ft/index.js]: Could we make this alt text a bit more descriptive, please? Maybe change to "image of an Agile stand up in progress"

File: site/pages/our-work/case-study/camden-market/index.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const CaseStudyCamdenMarket = ({ contactUsURL }: CaseStudyCamdenMarketProps) =>
       <Shapes />
       <div className={styles.header__container}>
         <div className={styles.header__imageContainer}>
-          <img src={headerImage} alt="shopping cart" className={styles.header__image} />
+          <img src={headerImage} alt="Camden Market website" className={styles.header__image} />
```
Comment (gregdorward) [inline site/pages/our-work/case-study/camden-market/index.js]: Change to "an image showing the Camden Market website on a mobile device"

**Comment by chris-brookes** [comment]
Technology page:
Same id is being used more than once, id="path-1"

Join-us page:
iframe does not have a title.

**Comment by chris-brookes** [comment]
@martinhartt - It looks like the first issue that @gregdorward entered (the multiple label issue), is only on the mobile view. The 'Menu' link and the 'Close' link (in the pop-up menu), both have '<label for="burger" '. The 'id="burger" ' should only be able to associate with one label.
I might not be 100% right, but I hope this will point you in the right direction.

File: site/pages/our-work/case-study/ft/index.js (hunk 1 above)
Comment (martinhartt) [inline site/pages/our-work/case-study/ft/index.js]: Sounds good, although the linter doesn't like the use of "image" in the alt-text so I'll put "Agile stand up in progress" instead if thats okay.

screen shot 2017-08-07 at 15 59 31

Comment (martinhartt) [inline site/pages/our-work/case-study/ft/index.js]: Also saw this on the link you included

screen shot 2017-08-07 at 16 10 33

Comment (gregdorward) [inline site/pages/our-work/case-study/ft/index.js]: haha good spot, sorry about that! In that case, you can ignore me there, maybe just update it to "An Agile stand up in progress"?

**Comment by martinhartt** [comment]
Seems that the 404 page isn't correctly deployed for each commit preview, which explains why there are so many issues which I thought I fixed. Once it is deployed to production, the issues should be resolved.

**Pull request merged**